aws/build.rake

```diff
@@ -94,7 +95,7 @@ def threaded_each(array, thread_count=2)
   threads.each(&:join)
 end
 
-if (rack_env?(:staging) && CDO.name == 'staging') || rack_env?(:development)
+if rack_env?(:staging) || rack_env?(:development)
```

Comment:
I don't think it changed. I think it's been a redundant check for some time, and when I saw it I was confused so I fixed it. Could change this separately if we want to reduce the change surface in this PR.